sor/src/main/java/com/bazaarvoice/emodb/sor/db/astyanax/StitchedRow.java

```diff
@@ -0,0 +1,357 @@
+package com.bazaarvoice.emodb.sor.db.astyanax;
+
+
+import com.datastax.driver.core.*;
+import com.google.common.reflect.TypeToken;
+import com.netflix.astyanax.serializers.StringSerializer;
+import com.netflix.astyanax.serializers.UUIDSerializer;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.net.InetAddress;
+import java.nio.ByteBuffer;
+import java.util.*;
+
+public class StitchedRow implements Row {
```

Comment:
This is pretty fragile code that just happens to work because of the limited calls that are being made to it.  Any slight change to its usage, such as re-arranging the columns or querying for a column slightly differently, can break it.

If you look at `ArrayBackedRow` from the CQL driver it may provide a template for a cleaner and less fragile implementation (it's private so it can't be instantiated or subclassed, which is too bad).